### PR TITLE
Fix translation of Extension Import Catalog Secret

### LINKED ISSUE #10205: UI Extension `Import Extension Catalog` has non-localised string
**Setup**
- Rancher version: 2.8.next1 dev time

**Describe the bug**
- UI Extension `Import Extension Catalog` has non-localised string

**To Reproduce**
- Nav to Extensions --> top right three dot menu --> `Manage Extension Catalogs` --> `Import Extension Catalog`

**Result**
- `imagePullSecret` label missing

**Screenshots**
image

**Additional context**
- caused via original translation changing to an object. see https://github.com/rancher/dashboard/pull/10028

### PR DESCRIPTION
**Pull request by richard-cox**
### Summary
Fixes #10205

### Occurred changes and/or fixed issues
- caused via original translation changing to an object. see https://github.com/rancher/dashboard/pull/10028
- give this translation it's own entry... and use the same tooltip

### Screenshot/Video
image

### CODE REVIEW

File: shell/assets/translations/en-us.yaml [hunk 1]
```diff
@@ -4134,6 +4134,9 @@ plugins:
         image:
           label: Catalog Image Reference
           placeholder: "e.g. hub.docker.io/example-org/my-image:latest"
+        imagePullSecret:
+          label: Image Pull Secret
```
Comment (jordojordo) [inline shell/assets/translations/en-us.yaml:4138]: Should it be "Image Pull Secrets"?

**Review by jordojordo — CHANGES_REQUESTED** [review]

File: shell/assets/translations/en-us.yaml (hunk 1 above)
Comment (richard-cox) [reply shell/assets/translations/en-us.yaml]: ahh, thought it was multiple, but the `:multiple="true"` was clear as day. good catch!

**Review by jordojordo — APPROVED** [review]

**Comment by nwmac** [comment]
@jordojordo @richard-cox How would we test this? Could we add automated tests? Is it worth it?

**Comment by richard-cox** [comment]
Nothing for this specific case. We could add something generic to `t` that could cause e2e tests to fail if the translation is missing, that would add value but scope creep for this issue

**Pull request merged**